ext/mbstring/libmbfl/mbfl/mbfl_allocators.c

```diff
@@ -33,11 +33,6 @@
 #endif
 
 #include <stdlib.h>
-
-#ifdef HAVE_MEMORY_H
```

Comment:
Also drop drop from config.h.in / config.h.w32?

Reply:
Thanks missed the one in the confi.h.w32. Solved in the next commit. The one in the config.h.in is being removed via the separate pull request https://github.com/php/php-src/pull/4143 where it is removed altogether actually since it is not needed. thank you.